man/man1/git-town.1

```diff
@@ -72,3 +74,9 @@ Add a new perennial branch
 
 .IP "git town perennial-branches --remove <branch_name>" 4
 Remove branch from perennial branches
+
+.IP "git town pull-branch-strategy" 4
+Displays your pull branch strategy
+
+.IP "git town pull-branch-strategy <strategy>" 4
+Set your pull branch strategy as <strategy>
```

Comment:
Set the given pull branch strategy for the current repository.
Allowed values are "merge" and "rebase".